config/initializers/sprockets-es6.rb

```diff
@@ -1,6 +1,8 @@
 # unfortunately there seems no other way to set the options because .call is called directly from sprockets
-Sprockets::ES6.instance.instance_variable_set(:@options, {:optional => %w(
```

Comment:
Won't be there for much longer, I was just waiting for a [new](https://github.com/TannerRogalsky/sprockets-es6/pull/20) `sprockets-es6` :).